getting-started.md

```diff
@@ -31,17 +31,60 @@ page_nav:
 
 This website helps people operating Tezos nodes to synchronize to the head of the chain, so they are operational faster.
 
-New snapshots are uploaded twice a day.
+New snapshots are uploaded twice a day. (Granada and Mainnet)
+
+We're are migrating to a new snapshot system where -
+
+Hangzhounet (currently deployed)
+
+* Rolling snapshots **every 30 seconds**
+
+* Rolling tarballs **every 30 seconds**
+
+* Archive tarballs **every 1 minute 18 seconds**
+
+Granada (planned)
+
+* Rolling snapshots **every 30 seconds**
+
+* Rolling tarballs **every 30 seconds**
+
+* Archive tarballs **every 1 minute 18 seconds**
+
+Mainnet (planned)
+
+* Rolling snapshots **every 60 seconds**
+
+* Rolling tarballs **every 60 seconds**
+
+* Archive tarballs **every 12 hours**
 
 This project was sponsored by a grant from the [Tezos foundation](https://tezos.foundation/).
 
+## What is a tarball ?
+
+Tezos snapshots can take a long time to import, especially for mainnet. 
+
+However, we simply tar up `/var/node/tezos` on our rolling and archive nodes (excluding `peers.json` and `identy.json`), lz4 it, and provide it for you you download.
+
+For example- you will have a fully synced mainnet rolling node in **9 minutes**.
+
+## How to use a tarball?
+
+For example to expand one of our tarballs for an **hangzhounet archive node** -
+
+```bash
+curl -LfsS "https://d1u3sv5wkszf4p.cloudfront.net/hangzhounet-archive-tarball" \
+| lz4 -d | tar -x -C "/var/tezos"
+```
+
```

Comment:
add a warning:

"While Tezos tarballs can help you bootstrap a fresh node faster, it does it at the expense of safety: unlike snapshot import, no sanity check is performed before starting the node. Please evaluate whether this is appropriate for your use case."